drivers/hmis/lib/form_data/default/fragments/current_living_situation.json

```
@@ -6,45 +6,135 @@
   "item": [
     {
```

The information date should be collected as user input
```suggestion
    {
      "type": "DATE",
      "link_id": "cls-information-date",
      "text": "Information Date",
      "mapping": { "field_name": "informationDate" },
      "required": true
    },
    {
```